src/components/Node/Node.tsx

```diff
@@ -42,15 +57,19 @@ const Node: FC<NodeProps & { ref: Ref<HTMLDivElement> }> = forwardRef<HTMLDivEle
     },
     ref
   ) => {
+    const classes = useStyles();
```

Comment:
I know material-ui's examples use the variable name `classes` for styles, but our style guide suggests the name `styles`.  `classes` is a bit more of an overloaded term and it's not immediately obvious that it's talking about _style_ classes specifically.

This is also the case in: `Navigation`, `PathwaysList`, and `PatientSnapshot`

Reply:
Agree with this, but since we're already using `styles`, we have a namespace issue. If we decide to merge all the styles together using material UI, we can have everything under `styles`. I didn't merge the stylesheets for these files for consistency, but we should revisit this and consider making a task to do so across the project.